backend/src/main/java/com/jobwait/domain/ValidEducationLevel.java

```diff
@@ -1,11 +1,26 @@
 package com.jobwait.domain;
 
+import com.fasterxml.jackson.annotation.JsonAlias;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
 public enum ValidEducationLevel {
+    @JsonProperty("Less than high school")
+    @JsonAlias("LESS_THAN_HIGHSCHOOL")
```

Comment:
@JsonProperty is for the deserialization (how it comes from frontend JSON)
@JsonAlias is for the serialization when it comes from db because otherwise it would try to use the property

I am not really sure how to make this nicer... its kind of fucked up....

Reply:
I'm not sure why you're doing it like this, why use aliases? Why not only accept LESS_THAN_HIGHSCHOOL in the request body, and store LESS_THAN_HIGHSCHOOL in the database, and return LESS_THAN_HIGHSCHOOL back to the user when fetched? We can have the frontend interpret LESS_THAN_HIGHSCHOOL as "less than high school" if we want it to be more human readable, no?

Reply:
yeah I guess thats fair, I can just do a mapping for the readable prompts on frontend -> what we send to the backend, I'll change this.

Reply:
I think communication between frontend and backend should not focus on human readable keys and instead focus on consistency and simplicity, i.e. it's more of an evil to use aliases like this than to just convert it on the frontend imo